Source/WebCore/layout/formattingContexts/inline/InlineFormattingContext.cpp

```diff
@@ -133,7 +133,7 @@ InlineLayoutResult InlineFormattingContext::layout(const ConstraintsForInlineCon
         return PreviousLine { lastLineIndex, { }, { }, { }, { } };
     };
 
-    if (root().style().textWrap() == TextWrap::Balance) {
+    if (root().style().textWrapMode() == TextWrapMode::Wrap && root().style().textWrapStyle() == TextWrapStyle::Balance) {
```

Comment:
I really wish we had a boolean for `text-wrap-mode` instead of an enumeration for call sites like this.

Reply:
We do have a helper method called `RenderStyle::autoWrap`, but I need to update it to check for `textWrapMode()` instead of calling a deprecated method. I'll do this in a follow up.